app/views/podcast_planner/_form_draft.html.erb

```diff
@@ -0,0 +1,68 @@
+<% data = {controller: "planner", planner_dates_count_value: @planner.dates&.count.to_i} %>
+<%= form_with(url: podcast_planner_path(@podcast), method: :post, data: data) do |form| %>
+
+  <div class="col-12 mb-4">
+    <div class="card shadow border-0">
+      <div class="card-header card-header-info d-flex justify-content-between">
+        <h5 class="card-title flex-grow-1"><%= t(".title.settings") %></h5>
+        <%= help_text t(".help.settings") %>
+      </div>
+
+      <div class="card-body pt-4 pb-0" data-morph="false">
```

Comment:
i'm assuming this `data-morph="false"` is to cancel out the morphdom controller that is above this partial, in order to utilize its own morphdom controller in the turbo frame below? does this mean you can't have a morphdom controller within a morphdom controller (like the parent still overrides the child or something)? or does it mean that we want to control the elements that get changed/updated within this element independently from changes made outside of this element? i still need to read up on morphdom as i still struggle to understand how to use it.

Reply:
Yes - this prevents the parent `turbo_frame_tag "podcast-planner"` (all the way up in `show.html.erb`) from blowing away any changes you've made to the selection.  Without this, every time you made changes in the calculator, it would reset these format/ad-breaks/time-of-day fields.

That `data-morph` thing is something I added in `morphdom_controller.js`.  As morphdom is traversing the DOM from top to bottom, if it hits any element with `data-morph=false`, it stops making updates for that element and its children.